IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2" Name="TypeOfArcExtinction" UniqueId="6c51b375-bc9b-4649-ae3c-f764aec5f982" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" />
```

Comment:
I am missing a Penum.

Reply:
Change to single value. Commit : b5e658cad21e4944423ed95d945268c722299c68

Reply:
Is the property required?